Dockerfile.nginx

```diff
@@ -0,0 +1,19 @@
+# ---------------------------------------------
+# Build static assets
+# ---------------------------------------------
+FROM node:8.12 as assets
+
+COPY *.js* ./
+COPY fala/assets-src ./fala/assets-src
+RUN npm install
+RUN ./node_modules/.bin/gulp build --production
+
+# ---------------------------------------------
+# Configure Nginx 
+# ---------------------------------------------
+FROM nginx:1.14-alpine
+COPY conf/nginx.conf /etc/nginx/nginx.conf
+RUN mkdir /home/static
```

Comment:
_nitpick_ An explicit `mkdir` is not strictly necessary as `COPY` will create a target directory if it's missing.